data-types.md

```diff
@@ -3,4 +3,408 @@ title: Data Types
 toc: false
 ---
 
-Coming soon.
\ No newline at end of file
+CockroachDB supports the followinig data types. Click a type for more details.
+
+Type | Description | Example
+-----|-------------|--------
+[`INT`](#int) | A 64-bit signed integer. | `12345`
+[`DECIMAL`](#decimal) | An exact, fixed-point number. | `DECIMAL '1.2345'`
+[`FLOAT`](#float) | An inexact, floating-point number. | `1.2345`
+[`BOOL`](#bool) | A Boolean value. | `true` 
+[`DATE`](#date) | Year, month, day. | `DATE '2016-01-25'`
+[`TIMESTAMP`](#timestamp) | A date and time pairing. | `TIMESTAMP '2016-01-25 10:10:10'`
+[`INTERVAL`](#interval) | A span of time. | `INTERVAL '2h30m30s'`
+[`STRING`](#string) | A string of characters. | `'a1b2c3'`
+[`BYTES`](#bytes) | A string of binary characters. | `'a1b2c3'`
+
+## `INT`
+
+The `INT` type stores 64-bit signed integers, that is, whole numbers from 9,223,372,036,854,775,808 to 9,223,372,036,854,775,807. 
+
+#### Synonyms 
+
+In CockroachDB, the following are synonyms of `INT` and are implemented identically: 
+
+- `SMALLINT` 
+- `INTEGER` 
+- `INT64` 
+- `BIGINT`
+- `BITS`
```

Comment:
Ah, this should be `BIT` instead. I see this in [types_test.go](https://github.com/cockroachdb/cockroach/blob/master/sql/parser/types_test.go#L30). And when I set a column to `bit`, it accepts that and converts it to `int`, just like the other synonyms:

```
root@:26257> create table bits (a bit, b int);
CREATE TABLE
root@:26257> show columns from bits;
+-------+------+-------+----------------+
| Field | Type | Null  |    Default     |
+-------+------+-------+----------------+
| a     | INT  | true  | NULL           |
| b     | INT  | true  | NULL           |
| rowid | INT  | false | unique_rowid() |
+-------+------+-------+----------------+
```

OK to leave this in as `BIT`?